microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -57,6 +57,7 @@ def ltp_lite(
         # parse variables
         tests = variables.get("ltp_test", "")
         skip_tests = variables.get("ltp_skip_test", "")
+        ltp_tests_git_tag = variables.get("ltp_tests_git_tag")
```

Comment:
Give it a default value, or it will fail if not specified.

Reply:
It would return None by default, right? Added "" similar to line 58,59

Reply:
It will fail if a value is not in dict. You can test it, after implemented.